### ISSUE TITLE
Affiliations

### ISSUE THREAD

**Issue by msdemlei** [ivoa-std/VOEvent#16]
In this document, the authors come with affiliations, as in

```
\author{\\Alasdair {\bf Allan}, University of Exeter, UK}
```

Ignoring for the moment the double backslash, this is certainly not the way the author macro was intended to work, and I forsee all kinds of trouble; the argument of \author should be an author name and nothing else.

In most of our documents, we don't give affiliations (actually, in none so far written in ivoatex).  If we want them sufficiently badly, we ought to somehow support that in ivoatex.  But at any rate they shouldn't be hacked in like that.

**Comment by BaptisteCecconi** [comment]
This was indeed a tacky way to reproduce the content of the MS Word version of version 2.0 of the recommendation. In the modern open science era, I think associating affiliation with authors should be possible, but more importantly, I think we should support ORCID for authors as soon as possible.

**Comment by msdemlei** [comment]
On Thu, Jun 22, 2023 at 01:11:42AM -0700, Baptiste Cecconi wrote:
> This was indeed a tacky way to reproduce the content of the MS Word
> version of version 2.0 of the recommendation. In the modern open
> science era, I think associating affiliation with authors should be
> possible, but more importantly, I think we should support ORCID for
> authors as soon as possible.

I'm kinda neutral on both (while I see ORCIDs may be useful, I'm
generally scared by machine-readable identifiers for humans) as long
as it's not a lot of work *for me*.

On ORCIDs: ivoatex already lets you associate URIs with persons; just
write \author[uri]{name}.  We've been using that for Twiki links so
far, but why not recommend putting ORCIDs in there (which, I
understand, these days are https URIs anyway)?  If you think that's
reasonable, could you imagine doing a PR against ivoatexDoc
recommending that practice?

As to affiliations, I'm not sure I get the connection to open
science -- how do they help there?

Anyway, *if* we want them, I'd say we should lift one of the
elaborate machineries that the journals have for them; if we start
fiddling ourselves, we'd probably end up at one of those after a long
toil anyway.  Neither is a nice prospect for me, I have to admit, so
I'd need either a PR against ivoatex or a serious amount of
additional convincing on why affiliations are a good thing *on IVOA
documents*.

**Comment by BaptisteCecconi** [comment]
This discussion should probably occur in a wider scope (e.g., interop@ivoa.net)

In the FAIR guidelines, it is recommended to included PIDs when possible for better _Findability_. In the DOI landing pages I manage, I set ORCIDs for persons, when available, ROR or Re3Data identifiers for affiliations, IVOA-UAT keywords, etc.

**Comment by Zarquan** [comment]
> This discussion should probably occur in a wider scope (e.g., [interop@ivoa.net](mailto:interop@ivoa.net))

Yep, agree. So for this version of VOEvent, should we remove the affiliations and come back to them later ?

**Comment by msdemlei** [comment]
On Thu, Jun 22, 2023 at 09:19:12AM -0700, Baptiste Cecconi wrote:
> This discussion should probably occur in a wider scope (e.g.,
> interop@)

interop@ is too wide, and this topic has a high bikeshedding
potential on top, so it's likely people will see 20 messages on a
single day in their mailbox and they'll be pi^H^Hnot amused.

We've sort of hijacked stdproc@ for this kind of thing.  But then,
for the affiliations we don't need a lot of discussion; nobody really
*dislikes* them.  The real problem is finding someone to
put some mechanism in to manage them -- if there's a patch, I don't
think anyone will object to having the *option* of putting them in
(I'd probably still be too lazy, as I can't see much utility in them
in IVOA standards; and that lack of utility also means I won't spend
much effort on getting them in either).

Meanwhile, I'd second @zarquan's proposal to just take them out this
time around.

> In the FAIR guidelines, it is recommended to included PIDs when
> possible for better _Findability_. In the DOI landing pages I
> manage, I set ORCIDs for persons, when available, ROR or Re3Data
> identifiers for affiliations, IVOA-UAT keywords, etc.

Fair enough (except I still think the benefit of person identifiers
in data or literature discovery isn't high enough to warrant
facilitating metrics and privacy breaches, but of course I'm fairly
alone in my skepticism against metrics).

So again: what if we recommended the URIs below the authors to be
ORCIDs?  We could do that without a single change to ivoatex, which I
like a lot...

**Comment by molinaro-m** [comment]
I am in favour of letting the __URI__ in `\author[uri]{name}` be an ORCID, that's a little more robust than the IVOA.TwikiNameSurname current usage (that can still live as a fallback).
